src/test/java/com/wikia/webdriver/common/core/CommonUtils.java

```diff
@@ -71,6 +71,7 @@ public static void createDirectory(String fileName) {
       new File(fileName).mkdir();
       System.out.println("directory " + fileName + " created");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)